pkg/retrieval/retrieval.go

```diff
@@ -143,7 +143,7 @@ func (s *Service) RetrieveChunk(ctx context.Context, addr, sourcePeerAddr swarm.
 		var (
 			peerAttempt  int
 			peersResults int
-			resultC      = make(chan retrievalResult)
+			resultC      = make(chan retrievalResult, 32)
```

Comment:
There is a retrieval rewrite in progress, I would hold on to this PR for now :)

Reply:
This PR changes only one line of code. It would be more beneficial to have leaks check enabled so rewrite would also benefit from same checks.